### DISCUSSION TITLE
What's wrong with the pins?

### DISCUSSION THREAD

**Discussion by shiwangyan** [logisim-evolution/logisim-evolution#1435] [General]
Pins do not show color
![QQ截图20220411214232](https://user-images.githubusercontent.com/60057517/162752798-a7a905e2-ce45-4f7e-91b3-fc616229f8cd.png)

**Comment by davidhutchens** [comment]
I think you have your preferences set for "Printer view". Open Preferences and check under Layout for a "Printer view" option. Turn it off. That should restore the colors you expect. Let us know if that works.
